sdk/keyvault/Azure.Security.KeyVault.Keys/src/JsonWebKey.cs

```diff
@@ -509,6 +509,8 @@ internal void WriteProperties(Utf8JsonWriter json, bool withId = false)
         void IJsonSerializable.WriteProperties(Utf8JsonWriter json) => WriteProperties(json);
 
         private static Func<RSAParameters, RSA> s_rsaFactory;
+
+        [System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessage("Trimming", "IL2026", Justification = "Incorrectly identifies RSA.Create(String); see https://github.com/Azure/azure-sdk-for-net/issues/40175 for discussion.")]
```

Comment:
Just an FYI - 

The pattern we typically follow is to make a separate method with just the call that we want to suppress the warning on. That way we don't over-suppress warnings (for example if someone would add some more incompatible code to this method, any new warnings would be suppressed and they wouldn't know about it.

```csharp
        private static RSA CreateRSAProvider(RSAParameters parameters)
        {
            if (s_rsaFactory is null)
            {
                // On Framework 4.7.2 and newer, to create the CNG implementation of RSA that supports RSA-OAEP-256, we need to create it with RSAParameters.
                [UnconditionalSuppressMessage("Trimming", "IL2026", Justification = "Incorrectly identifies RSA.Create(String); see https://github.com/Azure/azure-sdk-for-net/issues/40175 for discussion.")]
                MethodInfo GetRsaCreateMethod() =>
                    typeof(RSA).GetMethod(nameof(RSA.Create), BindingFlags.Public | BindingFlags.Static, null, new[] { typeof(RSAParameters) }, null);

                MethodInfo createMethod = GetRsaCreateMethod();
                if (createMethod != null)
                {
                    s_rsaFactory = (Func<RSAParameters, RSA>)createMethod.CreateDelegate(typeof(Func<RSAParameters, RSA>));
                }
                else
                {
                    s_rsaFactory = p =>
                    {
                        // On Framework, this will not support RSA-OAEP-256 padding.
                        RSA rsa = RSA.Create();
                        rsa.ImportParameters(parameters);
                        return rsa;
                    };
                }
            }
            return s_rsaFactory(parameters);
        }
```

Reply:
Thanks. I'll make a change to set a precedence. I didn't figure anyone would do something like that in a fairly targeted method already, but it's not impossible.